omod/src/main/java/org/openmrs/module/spa/web/controller/FrontendSPAController.java

```diff
@@ -0,0 +1,87 @@
+/**
```

Comment:
Please remove all classes that are not explicitly needed.  The SDK creates a bunch of "helpful" base classes for a common use case, but these should be deleted before committing if they are not needed.  Was this class created deliberately?